app/src/organisms/LabwarePositionCheck/LPCFlows/hooks/useLPCLabwareInfo/getLPCLabwareInfoFrom/index.ts

```diff
@@ -57,6 +57,15 @@ function getLabwareInfoRecords(
 
   params.lwLocInfo.forEach(combo => {
     const uri = combo.definitionUri
+
+    const matchedDef = params.labwareDefs?.find(
+      def => getLabwareDefURI(def) === uri
+    ) as LabwareDefinition | undefined
```

Comment:
```suggestion
    )
```
I think we can drop the type assertion here?